pages/register.vue

```diff
@@ -59,8 +75,12 @@ async function submit() {
           type="password"
           class="form-control"
           placeholder="Enter Password"
+          @blur="v$.password.$touch"
         />
       </div>
+      <div v-for="error of v$.password.$errors" :key="error.$uid" class="input-errors">
+        <div class="error-msg">{{ error.$message }}</div>
+      </div>
       <button html-type="submit" class="btn btn-primary" :disabled="isRegisterLoading" @click.prevent="submit">
```

Comment:
Users can submit the empty forms during login/register user.  Have to add validation when trying to submit empty form.